content/docs/components/serving/tfserving_new.md

```diff
@@ -6,32 +6,150 @@ weight = 51
+  namespace: kubeflow
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: mnist
+        version: v1
+    spec:
+      containers:
+      - args:
+        - --port=9000
+        - --rest_api_port=8500
+        - --model_name=mnist
+        - --model_base_path=gs://kubeflow-examples-data/mnist
+        command:
+        - /usr/bin/tensorflow_model_server
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /secret/gcp-credentials/user-gcp-sa.json
+        image: tensorflow/serving:1.11.1-gpu
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          initialDelaySeconds: 30
+          periodSeconds: 30
+          tcpSocket:
+            port: 9000
+        name: mnist
+        ports:
+        - containerPort: 9000
+        - containerPort: 8500
+        resources:
+          limits:
+            cpu: "4"
+            memory: 4Gi
+            nvidia.com/gpu: 1
+          requests:
+            cpu: "1"
+            memory: 1Gi
+        volumeMounts:
+        - mountPath: /var/config/
+          name: config-volume
+        - mountPath: /secret/gcp-credentials
+          name: gcp-credentials
+      volumes:
+      - configMap:
+          name: mnist-v1-config
+        name: config-volume
+      - name: gcp-credentials
+        secret:
+          secretName: user-gcp-sa
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  labels:
+    ksonnet.io/component: mnist-service
+  name: mnist-service
+  namespace: kubeflow
+spec:
+  host: mnist-service
+  subsets:
+  - labels:
+      version: v1
+    name: v1
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  labels:
+    ksonnet.io/component: mnist-service
+  name: mnist-service
+  namespace: kubeflow
+spec:
+  gateways:
+  - kubeflow-gateway
+  hosts:
+  - '*'
+  http:
+  - match:
+    - method:
+        exact: POST
+      uri:
+        prefix: /tfserving/models/mnist
+    rewrite:
+      uri: /v1/models/mnist:predict
+    route:
+    - destination:
+        host: mnist-service
+        port:
+          number: 8500
+        subset: v1
+      weight: 100
 ```
 
-Generate the deployment(version) component
+There are several places we can customize:
 
-```
-MODEL_COMPONENT=mnist-v1
-ks generate tf-serving-deployment-gcp ${MODEL_COMPONENT}
-ks param set ${MODEL_COMPONENT} modelName mnist
-ks param set ${MODEL_COMPONENT} versionName v1   // optional, it's the default value
-ks param set ${MODEL_COMPONENT} modelBasePath gs://kubeflow-examples-data/mnist
-ks param set ${MODEL_COMPONENT} gcpCredentialSecretName user-gcp-sa
-ks param set ${MODEL_COMPONENT} injectIstio true   // If you want to use istio
-```
+1. In the deployment resource, the container arg: `model_base_path` points to the model.
+   Change the value to your own model. 
+
+1. There are 3 parts for GCS access: volumes (secret `user-gcp-sa`), volumeMounts, and
```

Comment:
Since this is the first time we mention GCS, expand the abbreviation so people know what we're talking about:

```
The example contains three configurations for Google Cloud Storage (GCS) access:
```